prqlc/bindings/prqlc-python/python/prqlc/__init__.py

```diff
@@ -0,0 +1,6 @@
+# ruff: noqa: F403, F405
+from .prqlc import *
+
+__doc__ = prqlc.__doc__
+if hasattr(prqlc, "__all__"):
+    __all__ = prqlc.__all__
```

Comment:
(FYI this file seems a bit inelegant, but am guessing as a result of the python bindings. We could add a TODO to clean up if we're not confident we need this indirection)

Reply:
When I added the `debug` submodule to the python bindings, I needed to update the type declarations as well... which led me towards needing to use Maturin's [mixed Rust/Python project](https://www.maturin.rs/project_layout#mixed-rustpython-project) layout. I then just copied Maturin's default `__init__.py` (that they provide when you have a pure Rust project) into that directory.

I am sure we could be more precise about the module init here, but given that it was working fine before (plus the type stubs are there, which are more descriptive anyway) I decided this was good for now. The TODO makes sense especially if more Python-native code will be added later.